### ISSUE TITLE
bufferutil build fails v2.2

### ISSUE THREAD

**Issue by mdakovac** [bigbluebutton/bigbluebutton#10972]
**Describe the bug**
bufferutil build fails when installing version 2.2

**Installation type**
Manually using apt-get install bigbluebutton

**Console output**
```
> bufferutil@4.0.1 install /usr/local/bigbluebutton/bbb-webrtc-sfu/node_modules/bufferutil
> node-gyp-build
make: Entering directory '/usr/local/bigbluebutton/bbb-webrtc-sfu/node_modules/bufferutil/build'
  CC(target) Release/obj.target/bufferutil/src/bufferutil.o
../src/bufferutil.c:3:22: fatal error: node_api.h: No such file or directory
compilation terminated.
bufferutil.target.mk:90: recipe for target 'Release/obj.target/bufferutil/src/bufferutil.o' failed
make: *** [Release/obj.target/bufferutil/src/bufferutil.o] Error 1
make: Leaving directory '/usr/local/bigbluebutton/bbb-webrtc-sfu/node_modules/bufferutil/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/share/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 4.15.0-1088-gcp
gyp ERR! command "/usr/bin/nodejs" "/usr/bin/node-gyp" "rebuild"
gyp ERR! cwd /usr/local/bigbluebutton/bbb-webrtc-sfu/node_modules/bufferutil
gyp ERR! node -v v4.2.6
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
npm ERR! Linux 4.15.0-1088-gcp
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "rebuild"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! bufferutil@4.0.1 install: `node-gyp-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bufferutil@4.0.1 install script 'node-gyp-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bufferutil package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bufferutil
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bufferutil
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /usr/local/bigbluebutton/bbb-webrtc-sfu/npm-debug.log
```

also attaching npm-debug.log
[npm-debug.log](https://github.com/bigbluebutton/bigbluebutton/files/5647766/npm-debug.log)

**Comment by uka-support** [comment]
Please use ```bbb-install.sh```for installation. It's the only supported installation method right now.

```
../src/bufferutil.c:3:22: fatal error: node_api.h: No such file or directory
```

Sounds like some dependency is missing (but that's only a wild guess).

**Comment by prlanzarin** [comment]
See if this is fixed in 2.2.31. I've tweaked the dependency that usually causes this.
Moreover, this is not a fatal error. Besides the ugly stack, everything should work as usual.
